warehouse/query-core/src/main/java/datawave/query/jexl/visitors/QueryModelVisitor.java

```diff
@@ -367,18 +397,14 @@ protected JexlNode expandBinaryNodeFromModel(JexlNode node, Object data) {
             }
             if (log.isTraceEnabled())
                 log.trace("expanded:" + PrintingVisitor.formattedQueryString(expanded));
-            return expanded;
-        } else if (1 == product.size()) {
```

Comment:
curious why we had this in here in the first place

Reply:
I am guessing that the JexlnodeFactory.createNodeTreeFromPairs initially did not handle the single case.